docs/source/en/_toctree.yml

```diff
@@ -577,6 +577,8 @@
         title: ResNet
       - local: model_doc/segformer
         title: SegFormer
+      - local: model_doc/seggpt
```

Comment:
Can be moved to the multimodal section

Reply:
But `SegGpt` is a purely vision model

Reply:
Ok, yeah up for philosophical debate haha, we also consider LayoutLM as a multimodal model for instance since it uses text + bounding boxes, this one uses images + segmentation masks as prompt, which could be considered multiple modalities. But no strong opinion on this one

Reply:
Haha, okay, so in HF's standards `SegGpt` would be considered a multimodal model? If so let's keep consistency